src/api/pubsub.js

```diff
@@ -0,0 +1,105 @@
+'use strict'
+
+const promisify = require('promisify-es6')
+const bs58 = require('bs58')
+var Base64 = require('js-base64').Base64
+var Stream = require('stream')
+var http = require('http')
+
+let activeSubscriptions = []
+
```

Comment:
I was trying to keep it to be as little mutations as possible, just overwriting the entire variable instead of mutating the existing one.

But if you feel really strongly about this, I can change it